plugins/outputs/exec/README.md

```diff
@@ -0,0 +1,24 @@
+# Exec Output Plugin
+
+Please also see: [Telegraf Output Data Formats](https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md)
+
+This plugin executes external programs with serialized metrics data passed on stdin, one metric per line.  
+The external program is free to do what it wants with the data.
+
+### Configuration:
+
+```toml
+[[outputs.exec]]
+  # Shell/command
+  # Full command line to executable with parameters.
+  command = "/usr/local/bin/telegraf-output --config /etc/telegraf-output.conf"
```

Comment:
Probably echo here or some other command that could show multiple arguments

Reply:
would be nice to document this README and the sample config similarly to Docker's entrypoint:

https://docs.docker.com/engine/reference/builder/#entrypoint